plugins/aws/cache_utils.go

```diff
@@ -0,0 +1,38 @@
+package aws
+
+import (
+	"fmt"
+
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/aws/aws-sdk-go-v2/aws"
+)
+
+const (
+	mfaCacheKeyID        = "sts-mfa"
+	assumeRoleCacheKeyID = "sts-assume-role"
+)
+
+// stsCacheWriter writes aws temp credentials to cache using the awsCacheKey
+type stsCacheWriter struct {
+	awsCacheKey string
+	cache       sdk.CacheOperations
+}
+
+func (c stsCacheWriter) persist(credentials aws.Credentials) error {
```

Comment:
Nit: Could be nicer to have these cache wrappers use the same terminology as the underlying cache: `Put`/`Get`.